packages/preview-middleware-client/types/sap.ui.rta.d.ts

```diff
@@ -81,6 +81,15 @@ declare module 'sap/ui/rta/command/CommandFactory' {
         flexSettings?: FlexSettings;
     }
 
+    export interface ExtendedFlexSettings extends FlexSettings {
+        /**
+         * Selector extension for custom fragment generation
+         */
+        selector?: {
+            templateName?: string;
+        };
+    }
```

Comment:
As I can see this is not needed anymore and should be removed. There are 4 references to this interface.

Reply:
Removed